### PR TITLE
Milestone one: Linter-config

### PR DESCRIPTION
**Pull request by davmba**
In this ### Linter Configuration
In this project, we:
- Added html file
- Added css file
- Added test file

### CODE REVIEW

File: .github/workflow/linters.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Skip to content
```
Comment (Ismailco) [inline .github/workflow/linters.yml]: - [ ] Linter should be set up correctly for this repository, as for now, the linters check doesn't work on Github, kindly fix this

File: .github/workflow/linters.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Security
+Insights
+linters-config/html-css/.github/workflows/linters.yml
+@sinansevgi
+sinansevgi Update webhint to version 7
+Latest commit 39a68f5 7 days ago
+ History
+ 7 contributors
+@SpaYco@MauricioRobayo@sinansevgi@Meltrust@MacCrazyman@nidalaa@bolah2009
+56 lines (53 sloc)  1.91 KB
```
Comment (Ismailco) [inline .github/workflow/linters.yml]: - [ ] You should not modify the linter config file, kindly restore the linters.yml to the original state so the linters are working correctly

File: test.md [hunk 3]
```diff
@@ -0,0 +1 @@
+this file is to be ignored
```
Comment (Ismailco) [inline test.md]: - [ ] test.md should not be in your Github repo, kindly remove it from GitHub

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @davmba,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @davmba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**